src/Database/Driver.php

```diff
@@ -268,36 +268,6 @@ public function rollbackTransaction(): bool
         return $this->_connection->rollBack();
     }
 
-    /**
-     * @inheritDoc
-     */
-    abstract public function releaseSavePointSQL($name): string;
-
-    /**
-     * @inheritDoc
-     */
-    abstract public function savePointSQL($name): string;
-
-    /**
-     * @inheritDoc
-     */
-    abstract public function rollbackSavePointSQL($name): string;
-
-    /**
-     * @inheritDoc
-     */
-    abstract public function disableForeignKeySQL(): string;
-
-    /**
-     * @inheritDoc
-     */
-    abstract public function enableForeignKeySQL(): string;
-
-    /**
-     * @inheritDoc
-     */
-    abstract public function supportsDynamicConstraints(): bool;
```

Comment:
Were these removed because they are also defined in the interface?

Reply:
Yes, I mentioned it a while ago in chat and remembered when going through the interface.